src/Jira/Api.php

```diff
@@ -281,6 +281,27 @@ public function addComment($issueKey, $params)
         return $this->api(self::REQUEST_POST, sprintf('/rest/api/2/issue/%s/comment', $issueKey), $params);
     }
 
+     /**
+     * get comments from a ticket
+     *
+     * issue key should be YOURPROJ-221
+     *
+     * @param $issueKey
+     * @param $params
```

Comment:
This DocBlock is malformed:

* no parameter data types
* no parameter description
* short method description doesn't start with capital letter